exotic/api/nested_linear_fitter.py

```diff
@@ -333,11 +335,12 @@ def plot_periodogram(self):
 
         ########################################
         # subtract first order solution from data and recompute periodogram
-        maxper = 50 # constrain second order solution to be less than 50 days
+        maxper = maxper2 # constrain second order solution to be less than 50 days
```

Comment:
should this comment be updated?

Reply:
removed the comment... the period grid should likely be hand tailored anyway based on need.. it's tough to choose a one size fits all range